test/system/workflow_executions_test.rb

```diff
@@ -495,4 +497,118 @@ class WorkflowExecutionsTest < ApplicationSystemTestCase
     assert_link I18n.t(:'workflow_executions.show.edit_button')
     assert_no_link I18n.t(:'workflow_executions.show.remove_button')
   end
+
+  test 'can successfully delete multiple workflows at once' do
+    error_workflow = workflow_executions(:irida_next_example_error)
+    canceled_workflow = workflow_executions(:irida_next_example_canceled)
+    visit workflow_executions_path
+
+    assert_selector 'h1', text: I18n.t(:'workflow_executions.index.title')
+
+    assert_text 'Displaying 20 items'
```

Comment:
Change the Displaying n items to use the viral pagy limit_component summary translation throughout this file